cli/src/main/java/dev/starfix/Starfix.java

```diff
@@ -55,31 +62,13 @@ public int cloneCmd(@Parameters(index = "0") String url) {
             throw new IllegalArgumentException("Not a valid URI for git repository");
         }
 
-        // Configuration identifiers
-        String clone_path = "";// Holds path to destination Where the Repository Must Be Clonned
-        String ide = ""; // Holds command for IDE to open upon
-
-        File configFile = getConfigFile();// Calling functiono to fetch config file
 
-        // Reading Configuration File
-        try {
-            if (!configFile.exists()) {
-                // Incase no config file exists
-                editConfig(); // Calling function that lets user to configure
-            }
-
-            // System.out.println("\nLoading configurations.....\n");
-            ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
-
-            Config config = mapper.readValue(configFile, Config.class);
-
-            ide = config.ide;
-            clone_path = config.clone_path;
-            if (ide == null || clone_path == null)
-                editConfig(); // Incase of absence of configuration in file Launch Config
+        if(ide==null) {
+            throw new IllegalArgumentException("Editor not specifed, please specify via --editor or use config to set it up.");
+        }
 
-        } catch (Exception e) {
-            e.printStackTrace();
+        if(clone_path==null) {
+            throw new IllegalArgumentException("Clone path not specifed, please specify via --clone-path or use config to set it up.");
         }
```

Comment:
```suggestion
         new CommandLine(new Starfix()).usage(System.out); // Will invoke Picocli Help
        }
```

Reply:
not following this suggested change - printing the long usage help when we know exactly what the user needs seems not good ? do please open sepaerate PR if you think there is still a need.